src/utils/fetchClient.ts

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+const BASE_URL = 'https://mate.academy/students-api';
+
+// returns a promise resolved after a given delay
```

Comment:
Remove comment. Try to write obvious code with meaningful variable and function naming
Fix all cases if present